FEXCore/Scripts/config_generator.py

```diff
@@ -217,6 +217,14 @@ def print_man_environment_tail():
     ],
     "''", True)
 
+    print_man_env_option(
+    "FEX_PORTABLE",
+    [
+    "Allows FEX to run without installation. Global locations for configuration and binfmt_misc are ignored. These files are instead read from <FEXInterpreterPath>/fex-emu/ by default."
+    "For further customization, the environment variables FEX_APP_CONFIG_LOCATION and FEX_APP_DATA_LOCATION can be used."
```

Comment:
I missed that these aren't new in this PR. Since they have their own sections, this can be even shorter:

```suggestion
    "For further customization, see FEX_APP_CONFIG_LOCATION and FEX_APP_DATA_LOCATION."
```